packages/components/src/creditCardPaymentForm/CreditCardPaymentForm.js

```diff
@@ -0,0 +1,148 @@
+  +onChangeMonthOfExpiry: string => void,
+  +onChangeSecurityCode: string => void,
+  +onChangeYearOfExpiry: string => void,
+  +onRedeemPromoCodePress: () => void,
+  +onSecurityCodeHelpPress: () => void,
+  +securityCode: ?string,
+  +yearOfExpiry: ?string,
+|};
+
+export default function CreditCardPaymentForm({
+  cardholdersName,
+  creditCardNumber,
+  isCreditCardSaved,
+  monthOfExpiry,
+  onChangeCardholdersName,
+  onChangeCreditCardNumber,
+  onChangeCreditCardSaveSwitch,
+  onChangeMonthOfExpiry,
+  onChangeSecurityCode,
+  onChangeYearOfExpiry,
+  securityCode,
+  yearOfExpiry,
+  onSecurityCodeHelpPress,
+}: Props) {
+  return (
+    <Card style={styles.card}>
+      <View style={styles.form}>
+        <HeaderWithIcon label="Payment" iconName="credit-card" />
+
+        <View style={[styles.line, styles.creditCardNumber]}>
+          <TextInput
+            autoCorrect={false}
+            label="Credit card number"
+            maxLength={16}
+            onChangeText={onChangeCreditCardNumber}
+            type="number"
+            value={creditCardNumber}
+          />
+        </View>
+        <View style={[styles.line, styles.row]}>
+          <View style={styles.expiryDate}>
+            <TextInput
+              autoCorrect={false}
+              label="Expiry date"
+              maxLength={2}
+              onChangeText={onChangeMonthOfExpiry}
+              placeholder="MM"
+              type="number"
+              value={monthOfExpiry}
+            />
+          </View>
+          <View style={styles.expiryDate}>
+            <TextInput
+              autoCorrect={false}
+              label=" "
+              maxLength={2}
+              onChangeText={onChangeYearOfExpiry}
+              placeholder="YY"
+              type="number"
+              value={yearOfExpiry}
+            />
+          </View>
+          <View style={styles.securityCode}>
+            <TextInput
+              autoCorrect={false}
+              label="Security code"
+              maxLength={3}
+              onChangeText={onChangeSecurityCode}
+              type="number"
+              value={securityCode}
+              onHelpPress={onSecurityCodeHelpPress}
+            />
+          </View>
+        </View>
+        <View style={[styles.line, styles.cardholdersName]}>
+          <TextInput
+            onChangeText={onChangeCardholdersName}
+            label="Cardholder's name"
+            autoCorrect={false}
+            value={cardholdersName}
+          />
+        </View>
+        <View style={[styles.line, styles.row, styles.creditCardSaveSwitch]}>
+          <Text>Save card for next time </Text>
+          <Switch
+            onValueChange={onChangeCreditCardSaveSwitch}
+            value={isCreditCardSaved}
+          />
+        </View>
+      </View>
+    </Card>
+  );
+}
+
+const styles = StyleSheet.create({
+  card: {
+    web: {
+      alignItems: 'center',
+    },
+    flex: 1,
+  },
+  form: {
```

Comment:
Form on web have some variable width (instead of 720) and thanks to that it slightly jumps on input focus

![input](https://user-images.githubusercontent.com/2660330/54435260-6204e100-4730-11e9-82ca-49e99900d235.gif)

Setting ```width: '100%'``` can probably help, like in `PassengerForm`.

I seems to be a repeating pattern so we can probably make something like `FormWrapper` component from it. Which will be just white View on mobile and Card on web. 
To get styling more closer to current kiwi.com website.

<img width="905" alt="Screenshot 2019-03-15 at 14 37 57" src="https://user-images.githubusercontent.com/2660330/54435623-359d9480-4731-11e9-857e-f7ed9c96ff20.png">

Reply:
Done